### ISSUE TITLE
Addon missing in Mozilla repository

### ISSUE THREAD

**Issue by my-pixels** [RostiMelk/ACF-Tools#37]
Hi, I reinstalled Firefox today and now I can't find the ACF-TOOLS extension anywhere to download. It also disappeared from the official repository.

Is it possible to send the installation package?

**Comment by adrien-robert** [comment]
Would like to know as well

**Comment by RostiMelk** [comment]
Will get it back up on Mozilla asap. 
Expect an update on this issue next week.

**Comment by my-pixels** [comment]
I managed to dig out the old XPI from the profile backup, but I would still appreciate the extension in the repository. Thank you

**Comment by giorgiodare** [comment]
+1

**Comment by txrris** [comment]
Please!

**Comment by sba** [comment]
yes, please! - installed new PC and now I'm without ACF Tools in FF 😢 ... and I don't want to switch to chrome

**Comment by Zaidcrowe** [comment]
This still seems to be missing - any eta on restoring it? Thanks so much!

**Comment by mortennh** [comment]
Not in the store - hope you doin ok!

**Comment by Sanabria** [comment]
I ran into the same issue and generated the Firefox version from source (first time doing this). I'm attaching it here for other folks who may need it.

[acf-tools-v3.0.2-firefox.zip](https://github.com/RostiMelk/ACF-Tools/files/8274832/acf-tools-v3.0.2-firefox.zip)

**Comment by goerlitzer** [comment]
@my-pixels: is it possible that you send me your XPI file?

**Comment by RostiMelk** [comment]
Hi! 

This ZIP contains the following: A compiled version of both the Chrome and Firefox version of ACF Tools
[ACF Tools compiled.zip](https://github.com/RostiMelk/ACF-Tools/files/8660923/ACF.Tools.compiled.zip)

Unfortunately Mozilla never re-reviewed the package after it was deleted. As I don't work that much with WordPress anymore, I don't have the time and resources to continue maintaining this extension. 

You are of course free to do whatever you wish with the code, whether it's forking it, submitting a PR (I'll be happy to review, and merge it) or even uploading it to Mozilla yourself. The code was written at a time when I was just starting off with web development, and is far from ideal or even readable. I apologise for the inconvenience. 

I suggest checking out Hookturn's alternative to ACF Tools' code generator: https://hookturn.io/downloads/acf-theme-code-pro/
